website/source/docs/agent/caching/index.html.md

```diff
@@ -13,69 +13,99 @@ description: |-
 
 Vault Agent Caching allows client-side caching of responses containing newly
 created tokens and responses containing leased secrets generated off of these
-newly created tokens. The renewals of the secrets that are cached are also
+newly created tokens. The renewals of the cached tokens and leases are also
 managed by the agent.
 
 ## Caching and Renewals
 
-Caching and renewals are managed by the agent only under these scenarios.
+Caching and renewals are managed by the agent only under these specific scenarios.
 
 1. Token creation requests are made through the agent and not directly to the
-   Vault server. Login endpoints from various auth methods also fall under this
-   category along with token creation endpoints of the token auth method.
+   Vault server. This means that any login operations performed using various
+   auth methods and invoking the token creation endpoints of the token auth
+   method via the agent will result in the authentication response getting
+   cached by the agent. Note that the child tokens will only be cached by the
+   agent if the parent token is already being managed by the agent (the
+   exception to this is when an agent unmanaged token is used to create an
```

Comment:
This exception could be a third and separate scenario...?

Reply:
Since the first point was about token creation requests, it made sense to mention it in here. But I could see how it could be confusing. Paraphrased it. LMK if it looks okay now.